pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -77,6 +77,8 @@ type PreEnqueueCheck func(pod *v1.Pod) bool
 type SchedulingQueue interface {
 	framework.PodNominator
 	Add(pod *v1.Pod) error
+	// MoveUpPods moves given pods to activeQ in a best-efforts manner.
+	MoveUpPods(pods []*v1.Pod)
```

Comment:
`MoveUpPods` may not be very easy to understand what he is going to do. Do you need to rename it like
`MoveAllToActiveQ`、`MoveAllToActiveQueue` or other better name？

Reply:
only the pods in the map will be moved, `MoveAll*` seems to me is a little confusion IMHO.

Reply:
`MoveToActiveQueue` sounds good to me.

Reply:
My old plan was to add the function to FrameworkHandle, and hesitated to expose the internal details (activeQ) to the name. But now I'm implementing in a lightweight approach, and MoveXYZ only shows up in ScheduingQueue, so yes, `MoveToActiveQueue` is fine.